calculator/src/main/scala/zio/calculator/App.scala

```diff
@@ -72,6 +72,5 @@ object CalculatorApp extends zio.App:
     raft.use(raft => {
       for 
         _ <- new HttpServer(raft).run.fork.when(memberId == MemberId("peer1"))
-        _ <- raft.run
```

Comment:
if you remove this, the above should not be forked

Reply:
I add inside the makeManaged with forkManaged.